src/pages/__snapshots__/PrintPage.test.tsx.snap

```diff
@@ -1,486 +1,591 @@
   line-height: 1.1;
   font-size: 1.5rem;
 }
 
-.c3 h2 {
+.c9 h2 {
   margin: 1.5rem 0 0.75rem;
   font-size: 1.25rem;
 }
 
-.c3 h3 {
+.c9 h3 {
   font-size: 1.17rem;
 }
 
-.c3 h4 {
+.c9 h4 {
   font-size: 1rem;
 }
 
-.c3 h5 {
+.c9 h5 {
   font-size: 0.9rem;
 }
 
-.c3 h3,
-.c3 h4,
-.c3 h5,
-.c3 p {
-  margin-top: 1rem;
-  margin-bottom: 1rem;
+.c9 h3,
+.c9 h4,
+.c9 h5,
+.c9 p {
+  margin-top: 0;
+  margin-bottom: 0;
 }
 
-.c3 h1 + h2 {
+.c9 h1 + h2 {
   margin-top: -0.75rem;
 }
 
-.c3 h1 + p,
-.c3 h2 + p {
+.c9 h1 + p,
+.c9 h2 + p {
   margin-top: -0.75rem;
 }
 
-.c3 h3 + p,
-.c3 h4 + p,
-.c3 h5 + p {
-  margin-top: -1rem;
+.c9 h3 + p,
+.c9 h4 + p,
+.c9 h5 + p {
+  margin-top: 0;
 }
 
-.c3 > :first-child {
+.c9 > :first-child {
   margin-top: 0;
 }
 
-.c3 > :last-child {
+.c9 > :last-child {
   margin-bottom: 0;
 }
 
-.c3 dl {
+.c9 dl {
   margin: 1rem 0;
 }
 
-.c4 {
-  max-width: 66ch;
-  line-height: 1.2;
-}
-
-.c4 h1 {
-  margin: 2rem 0 1rem;
-  line-height: 1.1;
-  font-size: 1.5rem;
+.c9 > h3 {
+  font-size: 0.875em;
+  font-weight: 400;
 }
 
-.c4 h2 {
-  margin: 1.5rem 0 0.75rem;
-  font-size: 1.25rem;
+@media (min-width:480px) {
+  .c2 {
+    line-height: 1.3;
+  }
 }
 
-.c4 h3 {
-  font-size: 1.17rem;
+@media print {
+  .c11 {
+    color: black;
+  }
 }
 
-.c4 h4 {
-  font-size: 1rem;
+@media screen {
+  .c0 {
+    display: none;
+  }
 }
 
-.c4 h5 {
-  font-size: 0.9rem;
-}
-
-.c4 h3,
-.c4 h4,
-.c4 h5,
-.c4 p {
-  margin-top: 1rem;
-  margin-bottom: 1rem;
-}
-
-.c4 h1 + h2 {
-  margin-top: -0.75rem;
-}
-
-.c4 h1 + p,
-.c4 h2 + p {
-  margin-top: -0.75rem;
-}
-
-.c4 h3 + p,
-.c4 h4 + p,
-.c4 h5 + p {
-  margin-top: -1rem;
-}
-
-.c4 > :first-child {
-  margin-top: 0;
-}
-
-.c4 > :last-child {
-  margin-bottom: 0;
-}
-
-.c4 dl {
-  margin: 1rem 0;
-}
-
-.c5 {
-  margin-left: -1em;
-  text-align: center;
-  white-space: nowrap;
-}
-
-.c5::before,
-.c5::after {
-  display: inline-block;
-  width: 0;
-  overflow: hidden;
-  vertical-align: bottom;
-  text-align: left;
-  content: '…';
-  -webkit-animation: loadingEllipsis steps(4,end) 2s infinite;
-  animation: loadingEllipsis steps(4,end) 2s infinite;
-}
-
-.c5::before {
-  color: transparent;
-}
-
-.c0 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-flex-direction: row;
-  -ms-flex-direction: row;
-  flex-direction: row;
-  height: 100%;
-}
-
-.c0 > nav {
-  -webkit-flex: 1;
-  -ms-flex: 1;
-  flex: 1;
-}
-
-.c0 > main {
-  -webkit-flex: 2;
-  -ms-flex: 2;
-  flex: 2;
-}
-
-@media (min-width:480px) {
-  .c3 {
-    line-height: 1.3;
-  }
-}
-
-@media (min-width:480px) {
-  .c4 {
-    line-height: 1.3;
-  }
-}
-
-@media print {
-  .c0 {
-    display: none;
-  }
+@media (min-width:480px) {
+  .c9 {
+    line-height: 1.3;
+  }
 }
 
 <div
+  aria-hidden="true"
   class="c0"
 >
-  <main
+  <div
     class="c1"
   >
     <div
-      class="c2"
+      class="c2 ballot-header-content"
+    >
+      <h2>
+        Unofficial TEST Ballot
```

Comment:
@beausmith it appears these snapshots were only capturing the "Printing official ballot" content and not the ballot itself. I fixed that by referring to `childNodes[1]` instead of `firstChild`.

Reply:
👍